.travis.yml

```diff
@@ -60,3 +57,8 @@ cache:
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
+  - git fetch --all && git config --global user.email "-" && git config --global user.name "-"
+    && ./gradlew run -Dargs="--since d1"
+  - chmod ugo+x ./config/travis/deploy-surge.sh
```

Comment:
Just want to clarify this -  there a reason we do not need to set the permissions for our script: `./config/travis/run-checks.sh`, but it's needed for this ?

Reply:
Good catch, it is not needed.